packages/e2e-test-utils-playwright/src/activate-plugin.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Activates an installed plugin.
+ *
+ * @this {import('./').TestUtils}
+ * @param {string} slug Plugin slug.
+ */
+export async function activatePlugin( slug ) {
+	await this.switchUserToAdmin();
+	await this.visitAdminPage( 'plugins.php' );
+	const disableLink = await this.page.$(
```

Comment:
It might make sense to create a small helper function instead of using `this.page.$` which does the following:

`const locatorExists = async (locator: Locator) => (await locator.waitFor({ timeout: 1_000 }).catch((e: Error) => e)) instanceof Error;
`
This waits up to 1 second to wait for the element to appear. `page.$` on the other hand checks it immediately. We should probably put some API like this into the core.

Reply:
This was copied pasted from the old util, we might want to refactor it using REST API to speed it up in the future though. I'd tend to not change this for now, but it's still a great suggestion and we should keep that in mind in the test specs 👍 .